src/actions/playersActions.js

```diff
@@ -12,10 +18,23 @@ type Action =
   | UpdatePlayersListAction;
 
 
-function updatePlayersList(): Action {
+function updatePlayersList(id): Action {
```

Comment:
what is id for?

Reply:
When a user is clicked in the users list, the id is used to remove the user from the playersList state.

Shape of state is an object like this:

playersList: {
  1: {
    id: 1,
    username: 'user1'
  },
  2: {
    id: 2,
    username: 'user2'
  },
}